CheckboxItem/CheckboxItem.module.less

```diff
@@ -29,6 +32,11 @@
 			align-items: center;
 			-webkit-margin-start: 0;
 			-webkit-margin-end: @agate-checkboxitem-icon-margin;
+
+          .checkbox {
+            height: @agate-checkboxitem-checkbox-size;
+            width: @agate-checkboxitem-checkbox-size;
```

Comment:
I don't think this is needed. Current checkbox size of 24px + 2*border-size(3px), is 30px, like in the GUI